master/Makefile

```diff
@@ -2,7 +2,7 @@
 SHELL := bash
 GENERATED := packaging/LICENSE
 GENERATION_INPUTS = ../schemas/gen.py $(shell find ./pkg/schemas/ -name '*.go') $(shell find ../schemas/expconf -name '*.json')
-MOCK_GENERATION_INPUTS = ./internal/sproto/task.go ./internal/db/database.go ./internal/command/authz_iface.go
+MOCK_GENERATION_INPUTS = ./internal/sproto/task.go ./internal/db/database.go ./internal/command/authz_iface.go ./internal/rm/resource_manager_iface.go ./internal/task/allocation_service_iface.go
```

Comment:
I agree. `moq` would fix this with inline gen comments.

Reply:
not gonna do this now, but if you want, i'll stamp it.

Reply:
Since we have a ticket for spiking moq, I'll just leave it. Sorry I wasn't clear: this was just meant to be an observation of something weird, not a "hey, you should fix this."